internal/pkg/docker/docker.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
```

Comment:
What do you think if we had a build pkg, with the ECR publishing and dockerbuilding? (It might give us a useful place to put the code for building docker-compose files in the future for local support.

Reply:
Are you saying put all the `ecr` and `docker` code into a single `build` package or move the `docker` and `ecr` packages into a `build` folder?

Reply:
I was thinking of arranging the ecr and dockr packages into a build package. I don't have a strong preference on whether the docker and ecr in their own packages, but putting them both under build/ might be nice.